test/unit/proxy_test.rb

```diff
@@ -629,12 +629,12 @@ def analytics
     # no existing config for staging (sandbox)
     refute proxy.pending_affecting_changes?
 
-    Timecop.travel(1.second.from_now) do
```

Comment:
This change is failing right now for Rails 4. I asked Hery and he said this change is because there was a work done to index some keys in sphinx, which was done after_commit. Rails 4 does not trigger after_commit in tests, but rails 5.0 does.